File: fizzbuzz.rb
#!/bin/env ruby
def fizzbuzz()
  for input_num in 1..100 do
    fizzbuzz_check = input_num % 15
    fizz_check = input_num % 3
    buzz_check = input_num % 5

    if fizzbuzz_check == 0 then
      printf("FizzBuzz\n")
    elsif fizz_check == 0 then
      printf("Fizz\n")
    elsif buzz_check == 0 then
      printf("Buzz\n")
    else
      printf("#{input_num}\n")
    end

  end
end

def main()
  fizzbuzz
end

main

File: fizz_buzz.py
#!/bin/env python
def fizz_buzz():
    for input_num in range(1,1000001):
        fizz_buzz_check = input_num % 15
        fizz_check     = input_num % 3
        buzz_check     = input_num % 5

        if fizz_buzz_check == 0:
            print('FizzBuzz')
        elif fizz_check == 0:
            print('Fizz')
        elif buzz_check == 0:
            print('Buzz')
        else:
            print(input_num)

if __name__ == "__main__":
    fizz_buzz()
